sound/soc/sof/ops.h

```diff
@@ -37,6 +37,14 @@ static inline int snd_sof_remove(struct snd_sof_dev *sdev)
 	return 0;
```

Comment:
In commit: "platform specific shutdown stuff" -> "platform specific actions at shutdown"